src/styles/main.scss

```diff
@@ -0,0 +1,51 @@
+$days: 31;
+$month-length: (28, 29, 30, 31);
+$start-day: (mon, thu, wed, thu, fri, sat, sun);
+$gap: 1px;
+$cell-size: 100px;
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+@for $i from 1 through $days {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+.calendar {
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * ($cell-size + $gap) - $gap;
```

Comment:
The calculation `7 * ($cell-size + $gap) - $gap` seems to assume a 7-column layout for the calendar. However, the code does not handle the starting day of the month. You may need to include logic to offset the first day based on the `$start-day` variable.